cdap-api/src/main/java/io/cdap/cdap/api/app/ChangeSummary.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2014-2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.api.app;
```

Comment:
So there is a `ChangeSummary` class in proto for the "response" change-summary that is to be returned. This is in the request - do we want to combine them?

Reply:
yes, we should avoid classes with the same name

Reply:
I'm a bit confused now, thought this was the object in the response.

If this isn't part of any http response, it can be an internal class in the appfabric module, though it would be good to name it differently than whatever is in proto.

Reply:
discussed offline, we have `ChangeSummaryRequest` and `ChangeSummaryResponse` in proto for now